homeassistant/components/awair/translations/bg.json

```diff
@@ -13,7 +13,7 @@
                     "email": "Email"
                 }
             },
-            "user": {
```

Comment:
Please remove all changes to `translations` folder from your PR to avoid merge conflicts.

Reply:
Yes, still working through the unresolved comments